src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -37,7 +37,7 @@
     'urllib3[secure]>=1.18',
     'xmltodict',
     'pyOpenSSL',
-    'vsts-cd-manager<1.0',
+    'vsts-cd-manager<1.10',
```

Comment:
why not use `==`?

Reply:
We have planned to do few more releases of vsts-cd-manager without any modifying azure-cli.  '<' will provide flexibility to release vsts-cd-manager, however if use == then we have to update azure-cli also with every new release of vsts-cd-manager

Reply:
Ok, this is not the right way to do dependency version control. If you will introduce breaking change to the package, you should bump the minor or major version. Otherwise, you bump the micro version.

For example
`1.0.0` -> `2.0.0` major change
`1.0.0` -> `1.1.0` contains breaking change
`1.0.0` -> `1.0.1` -> `1.0.2` ... bug fixes, minor version, downstream packages don't need to be updated.

The idea that leaves 10 versions in between is not stable. So, using `<` is fitting, but it should be like:
```
'vsts-cd-manager<1.1.0` 
``

Reply:
Some reference:
1) http://semver.org/
2) https://www.python.org/dev/peps/pep-0440/

Reply:
Agreed. Great. I was looking for this!